doc/update065.md

```diff
@@ -0,0 +1,25 @@
+### dPoW 0.6.5 update information
+
+- On your 3P node, update your VRSC Coin's codebase to [7f971ed](https://github.com/VerusCoin/VerusCoin/tree/7f971ed3d4c5e1ec45737f374b9467b1a24d2679), build it and then restart it
+
+```bash
+# Build VRSC
+
+cd ~/VerusCoin/
+git pull
+git checkout 7f971ed3d4c5e1ec45737f374b9467b1a24d2679
+make -j4
+source ~/komodo/src/pubkey.txt
+./verusd -pubkey=$pubkey &
```

Comment:
rm, wrong folder and its in the next bit anyway

Reply:
updated